docs/application/get-started/mobile.md

```diff
@@ -1,44 +1,49 @@
-# Get Started for Mobile
+# Get Started with Tizen Mobile Applications
 
-The Get Started Tutorial teaches you how to create basic Tizen applications for mobile devices.
+Tizen ecosystem is enhanced with advanced Tizen-based smartphone application development.  
+For mobile devices, Tizen supports three primary application types, .NET, Web, and Native applications.
 
-Tizen supports three primary application types for mobile:
-
--   [.NET application](#dotnet-application) is a new way to develop applications for the Tizen operating system using C# language.
-
--   [Web application](#web-application) is a Web site stored on your device and built using Web-native languages, such as HTML5, CSS, and JavaScript. The Web application uses the Tizen Web framework to interact with the native subsystems.
-
--   [Native application](#native-application) is developed using C language. Developer can use the dozens of native API modules that cover a large range of capabilities such as camera, GPS, and accelerometer in addition to more advanced system settings and functionality.
-
-<a name="dotnet-application"> </a>
-## .NET Application
-
-The following guide demonstrates how to create and run a basic Tizen .NET application, which displays some text on the screen with no user interaction:
+## Mobile .NET Applications
```

Comment:
This guide is for Mobile devices. 
Is "Mobile" in headings redundant?  
When you change headings, please do it for all.

Reply:
Thank you for the comment.
I agree with you, will remove Mobile from the heading.
I will apply the same for all the headings. I will also update the other two files based on this comment.